<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 10, 10);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">;
background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-color: rgb(50, 50, 50);
border-style: groove;
border-width: 2px;
border-radius: 12px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="exploreProject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border-style: none;
background-color: rgb(50, 50, 50);
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Explore the GaelloX Project</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">;
background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-color: rgb(50, 50, 50);
border-style: groove;
border-width: 2px;
border-radius: 12px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="kickstart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border-style: none;
background-color: rgb(50, 50, 50);
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Get a hint of how to kickstart</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>20</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">;
background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-color: rgb(50, 50, 50);
border-style: groove;
border-width: 2px;
border-radius: 12px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="insights">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border-style: none;
background-color: rgb(50, 50, 50);
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Fetch Janine's main insights</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="insightsFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 12px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 210), stop:1 rgba(255, 255, 0, 238));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="insightLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>340</y>
       <width>580</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 200, 200);
border:  none;
background-color: rgb(10, 10, 10);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Access Janine's insights for you.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;How multiple events link altoghether to produce a specific trends on the market. &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Or how an asset has  been influenced  by events that look unrelated.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;More information can be gatheed in the chat section.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="insightsWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="communityFrame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 12px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 210), stop:1 rgba(255, 255, 0, 238));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="communityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>580</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 200, 200);
border:  none;
background-color: rgb(10, 10, 10);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;The Gaello X project beleives in thepower of Community.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Virtually meet like-minded folks and bullet-proof your startegies alongside with Janine.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Or just seek for help if you need some.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="communityWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="plusFrame">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>260</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 12px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 210), stop:1 rgba(255, 255, 0, 238));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="plusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>580</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 200, 200);
border:  none;
background-color: rgb(10, 10, 10);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;The adventure does not stop here!&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Explore more incoming projects from the GaelloX creators.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;No doubt you will find something that catches your interest!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="plusWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>580</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images/backgrounds.qrc"/>
  <include location="../rsrc/images/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
